src/ModelStorage.jl

```diff
@@ -1,4 +1,70 @@
 module ModelStorage
+using JSON3
+using Stipple
+import Stipple: INTERNALFIELDS, AUTOFIELDS, Reactive
+
+const DEFAULT_EXCLUDE = vcat(INTERNALFIELDS, AUTOFIELDS)
+
+"""
+    model_values(model::M; fields::Vector{Symbol} = Symbol[], exclude::Vector{Symbol} = Symbol[], json::Bool = false) where M
+
+Exports the values of reactive fields from a Stipple model. Returns either a Dict of field-value pairs or a JSON string
+if json=true.
+
+### Example
+
+    @app TestApp2 begin
+        @in i = 100
+        @out s = "Hello"
+        @private x = 4
+    end
+
+    model = @init TestApp2
+    exported_values = Stipple.ModelStorage.model_values(model)
+"""
+function model_values(model::M; fields::Vector{Symbol} = Symbol[], exclude::Vector{Symbol} = Symbol[], json::Bool = false) where M
+  field_list = isempty(fields) ? fieldnames(M) : fields
+  excluded_fields = vcat(DEFAULT_EXCLUDE, exclude)
+
+  field_dict = Dict(field => getfield(model, field)[] for field in field_list 
+                  if field ∉ excluded_fields && getfield(model, field) isa Stipple.Reactive)
+
+  json ? JSON3.write(field_dict) : field_dict
+end
+
+"""
+    load_model_values!(model::M, values::Dict{Symbol, Any}) where M
+    load_model_values!(model::M, values::String) where M
+
+Loads values into the fields of a ReactiveModel. Accepts either a Dict of field-value pairs or a JSON string.
+
+### Example
+
+    values_dict = Dict(:i => 20, :s => "world", :x => 5)
+    Stipple.ModelStorage.load_model_values!(model, values_dict)
+"""
+function load_model_values!(model::M, values::Dict{Symbol, Any}) where M
+  model_field_list = fieldnames(M)
+  excluded_fields = DEFAULT_EXCLUDE
+
+  for (field, value) in values
+    if field ∉ excluded_fields && field ∈ model_field_list
+      model_field = getfield(model, field)
+
+      if model_field isa Reactive
+        model_field[] = value
+      else 
+        @warn "Field $field is not reactive, skipping."
```

Comment:
@PGimenez I'm not sure we should show this warning, it will only confuse the users, you've seen how it goes...

Reply:
right. Also, the `excluded_fields`variable is already excluding the non-reactive fields of the `M`model, so one can only load reactive fields anyway. I'll remove it